check_manager/src/check_api.py

```diff
@@ -322,6 +323,18 @@ async def remove_check(
     return await check_backend.remove_check(auth_obj, check_id)
 
 
+@router.post(
+    RUN_CHECK_PATH,
+    status_code=status.HTTP_200_OK,
+    response_model_exclude_unset=True,
+)
+async def run_check(
+    response: Response, check_id: Annotated[CheckId, Path()]
+) -> None:
+    response.headers["Allow"] = "GET,POST"
```

Comment:
It seems to me this path only allows POST request
```suggestion
    response.headers["Allow"] = "POST"
```